CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * [ENHANCEMENT] Index Cache: Multi level cache adds config `max_backfill_items` to cap max items to backfill per async operation. #5686
 * [ENHANCEMENT] Query Frontend: Log number of split queries in `query stats` log. #5703
 * [ENHANCEMENT] Logging: Added new options for logging HTTP request headers: `-server.log-request-headers` enables logging HTTP request headers, `-server.log-request-headers-exclude-list` allows users to specify headers which should not be logged. #5744
+* [ENHANCEMENT] Query Frontend/Scheduler: Time check in query priority now considers range selectors, modifiers and lookback delta. #5758
```

Comment:
I prefer to just call it `data select time` or `data fetched time` here since it is hard to cover everything in promql that might affect select time. 
But it is up to you. If we want to mention those syntaxes, let's also add offset here. https://prometheus.io/docs/prometheus/latest/querying/basics/#offset-modifier